servicecontrol/upgrades/3to4/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Upgrade ServiceControl from Version 3 to Version 4
+summary: Instructions on how to upgrade ServiceControl from version 3 to 4
+isUpgradeGuide: true
+---
+
+## Overview
+
+Upgrading ServiceControl from version 3 to version 4 is a major upgrade and requires careful planning. During the upgrade process, the instance of ServiceControl that is being upgraded will no longer be available and will not be ingesting any messages.
+
+## Prerequisites
+
+Before upgrading to ServiceControl version 4 the instance being upgraded must be upgraded to at least [version 3.8.2](https://github.com/Particular/ServiceControl/releases/tag/3.8.2).
+
+For more information how to upgrade from Version 1.x to 3.8.2 consult the [upgrade documentation](/servicecontrol/upgrades/).
+
+## ServiceControl Audit
+
+ServiceControl version 4 introduces a new separate process to manage the audit queue called a ServiceControl Audit instance. This instance reads messages from the audit queue, stores them in it's internal database, and (optionally) forwards the processed messages to an audit log queue.
+
+The original ServiceControl instance will no longer manage the audit queue. It can still contain audit messages that have already been read from the audit queue. These messages will be retained until the configured audit retention period has passed.
+
+This split is transparent to the other components of the Particular Software Platform, which should continue to connect to the main ServiceControl instance. All queries to the main ServiceControl instance will contain results from the Audit instance as well.
+
+When upgrading a ServiceControl instance to version 4, if it is configured to manage an audit queue, a new ServiceControl Audit instance will be created as a part of the upgrade process. A user will need to supply additional information about the new ServiceControl Audit instance.
+
+<Details on what that looks like in SMCU>
+
+NOTE: If the ServiceControl instance being upgraded is not configured to manage an audit queue (by setting the audit queue name to `!disable`), then no new ServiceControl Audit instance will be created.
+
+### Upgrading with PowerShell
+
+Use the `Invoke-ServiceControlInstanceUpgrade` PowerShell cmdlet to  upgrade an existing ServiceControl instance to version 4.
```

Comment:
Should we link here to the powershell doco you also created?

Reply:
On re-reading this, I'm not sure that the powershell doco page says anything different.